recipes/libvdwxc/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "libvdwxc" %}
+{% set version = '0.4.0' %}
+{% set build_number = 0 %}
+{% set mpi = mpi or 'nompi' %}
+
+{% if mpi != 'nompi' %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+{% else %}
+  {% set mpi_prefix = "nompi" %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://launchpad.net/{{ name }}/stable/{{ version }}/+download/{{ name }}-{{ version }}.tar.gz
+  sha256: 3524feb5bb2be86b4688f71653502146b181e66f3f75b8bdaf23dd1ae4a56b33
+
+build:
+  skip: True  # [not unix]
+  number: {{ build_number }}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build_number }}"
+  run_exports:
+    - {{ pin_compatible('numpy', min_pin='x.x', max_pin='x.x') }}
+    {% if mpi != 'nompi' %}
+    - {{ name }} * {{ mpi_prefix }}_*
+    {% endif %}
+  script:
+    - mkdir build && cd build
+    - ../configure --prefix={{ PREFIX }} CC=$CC FC=$FC CFLAGS="$CFLAGS -O3 -ffast-math -funroll-loops" FCFLAGS="$FCFLAGS -O3 -ffast-math -funroll-loops" --with-fftw3={{ PREFIX }}  # [mpi == 'nompi']
+    - ../configure --prefix={{ PREFIX }} CC=mpicc FC=mpifort CFLAGS="$CFLAGS -O3 -ffast-math -funroll-loops" FCFLAGS="$CFLAGS -O3 -ffast-math -funroll-loops" --with-fftw3={{ PREFIX }} --with-mpi={{ PREFIX }}  # [mpi != 'nompi']
+    - make -j$CPU_COUNT
+    - make check
+    - make install
+    - rm -rf {{ PREFIX }}/bin/libvdw_*test
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ mpi }}-mpicc  # [mpi != 'nompi']
+    - {{ mpi }}-mpifort  # [mpi != 'nompi']
+    - make  # [unix]
+  host:
+    - {{ mpi }}  # [mpi != 'nompi']
+    - fftw * {{ mpi_prefix }}_*
+
+
+test:
+  requires:
+    - python >= 3
+  commands:
+    - test -f ${PREFIX}/lib/libvdwxc.a || exit 1  # [unix]
```

Comment:
static libraries are discouraged. Is it possible to build shared libraries?

Reply:
Shared libraries **are** already generated too so should I split this package into `libvdwxc` and `libvdwxc-static` then?

Reply:
Or just remove the static library? Do you need it?

Reply:
The topic is rather complex. For the time being, I'll remove them and if users are really adamant about getting static libs, I'll create a static lib variant.